session/handler.go

```diff
@@ -153,6 +160,42 @@ func (h *Handler) whoami(w http.ResponseWriter, r *http.Request, ps httprouter.P
 	h.r.Writer().Write(w, r, s)
 }
 
+// swagger:parameters adminLogoutIdentity
+// nolint:deadcode,unused
+type adminLogoutIdentity struct {
+	// ID is the identity's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+}
+
+// swagger:route DELETE /sessions/identity/{id} v0alpha1 adminLogoutIdentity
+//
+// Calling this endpoint irrecoverably and permanently Invalidates all sessions tha belongs to a given Identity.
+//
+// This endpoint is useful for:
+//
+// - To forcefully logout Identity from all devices and sessions
+//
+//     Schemes: http, https
+//
+//     Security:
+//       oryAccessToken:
+//
+//     Responses:
+//       202: emptyResponse
+//       401: jsonError
+//       500: jsonError
+func (h *Handler) logout(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	if err := h.r.SessionPersister().DeleteSessionsByIdentity(r.Context(), x.ParseUUID(ps.ByName("id"))); err != nil {
```

Comment:
`x.ParseUUID` will panic (I think) on error. As the parameter comes from the outside, we cannot assume that it is valid, so instead parse the UUID with error handling:
```suggestion
	iID, err := uuid.FromString(ps.ByName("id"))
	if err != nil {
		h.r.Writer().WriteError(w, r, herodot.ErrBadRequest.WithError(err.Error())).WithDebug("...something a bit useful")
	}
	if err := h.r.SessionPersister().DeleteSessionsByIdentity(r.Context(), iID); err != nil {
```

Reply:
Also, what does the persister return in case of unknown identity or no session? I think it should still return `http.StatusAccepted` in those cases because the user is logged out, so the desired outcome is reached.